qcodes/instrument_drivers/Keithley/_Keithley_2600.py

```diff
@@ -626,6 +630,13 @@ def doFastSweep(self, start: float, stop: float, steps: int, mode: str) -> DataS
                 'VI' (current sweep two probe setup) or
                 'VIfourprobe' (current sweep four probe setup)
         """
+        try:
+            from qcodes_loop.measure import Measure
```

Comment:
Deprecate this method ?

Reply:
Not yet. This would require a bigger change https://github.com/QCoDeS/Qcodes/issues/4976